src/js/tabs/pid_tuning.js

```diff
@@ -181,10 +181,10 @@ TABS.pid_tuning.initialize = function (callback) {
             antiGravitySwitch.change(function() {
                 var checked = $(this).is(':checked');
                 if (checked) {
-                    const MAX_ACCELERATOR_GAIN = semver.gte(CONFIG.apiVersion, API_VERSION_1_43) ? 3.5 : 1.1;
-                    const itermAcceleratorGain = Math.max(ADVANCED_TUNING.itermAcceleratorGain / 1000);
-                    if (ADVANCED_TUNING.itermAcceleratorGain == 1000) {
-                        $('.antigravity input[name="itermAcceleratorGain"]').val(MAX_ACCELERATOR_GAIN);
+                    const DEFAULT_ACCELERATOR_GAIN = semver.gte(CONFIG.apiVersion, API_VERSION_1_43) ? 3.5 : 1.1;
+                    const itermAcceleratorGain = (ADVANCED_TUNING.itermAcceleratorGain / 1000);
+                    if (ADVANCED_TUNING.itermAcceleratorGain === 1000) {
```

Comment:
Sorry to not see this in the first review ;) 
After removing the `Math.max`, the `DEFAULT_ACCELERATOR_GAIN` is only used inside the if, so better to move inside it:
```javascript
 if (ADVANCED_TUNING.itermAcceleratorGain === 1000) {
   const DEFAULT_ACCELERATOR_GAIN = semver.gte(CONFIG.apiVersion, API_VERSION_1_43) ? 3.5 : 1.1;
   $('.antigravity input[name="itermAcceleratorGain"]').val(DEFAULT_ACCELERATOR_GAIN);
} else {
```

Another solution is to change the complete if-else by a:
```javascript
$('.antigravity input[name="itermAcceleratorGain"]').val(ADVANCED_TUNING.itermAcceleratorGain === 1000 ?  DEFAULT_ACCELERATOR_GAIN : itermAcceleratorGain;
```

But maybe in this case is clearer with the if-else.

Take the form that you prefer...

Reply:
dont worry,fixed now, i think that I prefer if/else in this case,as you said looks more clearer.

Reply:
It happens the same with the `itermAcceleratorGain` :( It is used only in the `else` part...

I think this confinement has made my slower in reactions...

Reply:
I was thinking exactly that for `itermAcceleratorGain`

Reply:
Yes, and it is modified too some lines up:

https://github.com/betaflight/betaflight-configurator/blob/12dfc4fd6038b8905cbe660e010e236ee55d578b/src/js/tabs/pid_tuning.js#L172

so we end assigning it twice. I think this line can be removed but is outside the scope of this PR. To me is OK now.

Reply:
@McGiverGim all is done now!